package.json

```diff
@@ -7,6 +7,17 @@
   "scripts": {
     "build": "yarn workspaces foreach run build",
     "lint": "yarn workspaces foreach run lint",
-    "test": "yarn workspaces foreach run test"
+    "test": "jest"
```

Comment:
I'd keep this and let every specific package to provide its own `test` script, which would execute `jest` CLI scoped to that package's directory.

Reply:
`yarn workspaces foreach run test` does make sense now that he set it all up again in each.

Reply:
Unfortunately that won't work, see https://github.com/openshift/dynamic-plugin-sdk/pull/31#discussion_r808949063